modules/core/core-api/src/main/java/com/enonic/xp/node/AggregatedNodeDataProcessor.java

```diff
@@ -0,0 +1,26 @@
+package com.enonic.xp.node;
+
+import java.util.List;
+
+import com.google.common.base.Preconditions;
+
+import com.enonic.xp.data.PropertyTree;
+
+public final class AggregatedNodeDataProcessor
```

Comment:
```suggestion
public final class ComposeteNodeDataProcessor
```

But do we really need it as an API? it should be be trivial to implement whenever needed